dev/breeze/src/airflow_breeze/commands/release_management_commands.py

```diff
@@ -451,7 +451,12 @@ def _check_sdist_to_wheel_dists(dists_info: tuple[DistributionPackageInfo, ...])
                     )
                     sys.exit(1)
                 pip_command = (python_path.__fspath__(), "-m", "pip")
-                run_command([*pip_command, "install", f"pip=={AIRFLOW_PIP_VERSION}"], check=True)
+                result = run_command(
+                    [*pip_command, "install", f"pip=={AIRFLOW_PIP_VERSION}"], check=False, stdout=DEVNULL
```

Comment:
Instead of devnull'ing stdout, can we capture it to show if things do go wrong? It could be helpful context to supplement stderr.

Reply:
Sure. Good idea. Done.